tests/phpunit/tests/blocks/editor.php

```diff
@@ -447,13 +447,19 @@ public function test_wp_get_post_content_block_attributes() {
 			),
 		);
 		// With no block theme, expect an empty array.
-		$this->assertSame( array(), wp_get_post_content_block_attributes() );
+		$this->assertSame( null, wp_get_post_content_block_attributes() );
```

Comment:
```suggestion
		$this->assertNull( wp_get_post_content_block_attributes() );
```

Let's use `assertNull` instead `assertSame`